src/style.css

```diff
@@ -1 +1,35 @@
 /* add styles here */
+
+/* okay */
+
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid using tag names for styling, except for `html` and `body`. Since the `body` tag is a special case where it is acceptable to set global styles, this is fine. However, ensure this is the only tag styled directly unless there's a specific reason to target another tag directly.